core-libs/setup/ssr/optimized-engine/ssr-optimization-options.ts

```diff
@@ -137,6 +137,7 @@ export enum RenderingStrategy {
 }
 
 export const defaultSsrOptimizationOptions: SsrOptimizationOptions = {
+  cacheSize: 3000, // 1.5GiB / 525KB (50% of minimal pod size 3GiB / 150% of OOTB page ~350KB)
```

Comment:
`should:` instead of a comment in this place, let's adjust the description of the jsdoc for this parameter. We can write there what is the default `cacheSize and how we defined such a value by describing a bit the calculations that have been made.

By doing it that way, customers will see all the important information when they want to modify this parameter in their applications.